server/lib/email.js

```diff
@@ -226,6 +226,7 @@ const isWhitelistedDomain = email => {
  */
 const generateEmailFromTemplate = (template, recipient, data = {}, options = {}) => {
   const slug = get(options, 'collective.slug') || get(data, 'collective.slug') || 'undefined';
+  const hostSlug = get(data, 'collective.slug');
```

Comment:
Shouldn't that be

```suggestion
  const hostSlug = get(data, 'collective.host.slug');
```

?

Reply:
@Betree Oops. Definitely !